### DISCUSSION TITLE
"Proxy" Chromium Browser

### DISCUSSION THREAD

**Discussion by hosykynam0619ch2008-ies** [DustinBrett/daedalOS#282] [Feature Requests]
Hi Dustin, I love your project! As right now you're still using iFrame for the browser :(, I know there's might be a lot of request asking you to use proxy as the chromium, but I know it's hard hosting proxy, anddd coding the browser btw ._., so I'm here to ask you if you could replace the browser with this: https://r.studyschooltoday.eu.org/ , I've found this website which is a proxy server but it's design to look exactly like chrome, which fits the deadalOS browser, it's save you time of making a browser, instead of building a browser, just embed this site as an iframe and enjoy! I'm sorry if this site not work for you, sooooooo.... maybe give it a try, I (*personaly*) think this design fits..... the browser.

Thank you for reading this, I appreciate your work 😊

**Comment by DustinBrett** [comment]
Hi and thank you for this suggestion. I will look into this proxy idea but for the most part I consider the Browser to be a novelty at this point and I am ok with the iframe limitation until a true connection level proxy can be added which is free and open and without ads or anything nefarious.

**Comment by Pixel556** [comment]
The website is a copy of the original and it always gets taken down by schools as it removes the website filter added on computers by schools.

You can view the original here: https://browser.rammerhead.org
